spec/season_spec.rb

```diff
@@ -43,4 +43,12 @@
     end
   end
 
+
+  describe "team accuracy" do
+    it "returns most accurate team" do
+      expect(@season.team_accuracy(20122013))
```

Comment:
you could remove line 49 as it doesn't actually test anything (we can do this in a future refactor of the whole test file when all the other methods are added)